ssl/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply from: "$rootDir/gradle/javaModule.gradle"
+
+archivesBaseName = 'ssl'
+group = 'io.crate'
+description = 'SSL Encryption for CrateDB'
+
+dependencies {
+    compile project(':core')
```

Comment:
If we want to use the elastic search`Loggers` class, we still have to include `es-core`.